cmd/attack/network.go

```diff
@@ -209,7 +209,7 @@ func NetworkDNSCommand(dep fx.Option, options *core.NetworkCommand) *cobra.Comma
 
 	cmd.Flags().StringVarP(&options.DNSServer, "dns-server", "", "123.123.123.123",
 		"update the DNS server in /etc/resolv.conf with this value")
-	cmd.Flags().StringVarP(&options.DNSHost, "dns-hostname", "H", "", "map this host to specified IP")
+	cmd.Flags().StringVarP(&options.DNSDomainName, "dns-domain-name", "d", "", "map this host to specified IP")
```

Comment:
Can we need to update the document?

Reply:
https://chaos-mesh.org/zh/docs/next/simulate-network-chaos-in-physical-nodes/#dns-%E6%95%85%E9%9A%9C%E7%9B%B8%E5%85%B3%E9%85%8D%E7%BD%AE%E8%AF%B4%E6%98%8E
Already update, and I will check all the updated config in the document later

Reply:
![image](https://user-images.githubusercontent.com/22956341/143372826-94739cbd-0c9b-4cd8-b32c-3c68c230dbaf.png)

Reply:
OK, I will fix it